umm-spec-lib/resources/json-schemas/umm/v1.8/umm-cmn-json-schema.json

```diff
@@ -1327,11 +1327,6 @@
       "type": "string",
       "enum": ["Direct Line", "Email", "Facebook", "Fax", "Mobile", "Modem", "Primary", "TDD/TTY Phone", "Telephone", "Twitter", "U.S. toll free", "Other"]
     },
-    "ISOTopicCategoryEnum": {
```

Comment:
I know we've had this discussion before and I forget what the consensus is, but I disagree with making changes to old versions of the schema